catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/content/brick/app/SpeakBrickTest.java

```diff
@@ -63,6 +69,15 @@ public void speakBrickTest() {
 
 		onBrickAtPosition(0).checkShowsText(R.string.brick_when_started);
 		onBrickAtPosition(brickPosition).checkShowsText(R.string.brick_speak);
+		onBrickAtPosition(brickPosition).checkShowsText(R.string.brick_speak_in_language);
+
+		onData(anything()).inAdapterView(withId(R.id.brick_speak_spinner)).atPosition(0).check(matches(withText(R
```

Comment:
create a wrapper for the locale spinner

Reply:
I've no idea how to create a wrapper for a such a spinner with "List<locale> as arrayAdapter" and "Strings(R.string.device_language + displayNames of the locales) as views".
by the way, as I did above works fine!